src/box/memtx_space.c

```diff
@@ -1121,21 +1135,32 @@ memtx_init_ephemeral_space(struct space *space)
 }
 
 /*
- * Ongoing index build state with statement used by
- * corresponding on_rollback triggers to prevent rollbacked changes appearance.
+ * Struct to allocate by memtx_build_on_replace trigger, contains
+ * transactional triggers with their data.
  */
-struct index_build_on_rollback_data {
+struct memtx_build_txn_triggers {
+	/** Link in memtx_ddl_state::txn_triggers. */
```

Comment:
Let's rename the struct to `memtx_build_stmt_trigger`:
- No 's' at the end so as not to confuse it with an array/list. It's basically one trigger object executed on transaction statement commit or rollback.
- `stmt` to emphasize that it's a transaction statement trigger.

Let's also rename `memtx_ddl_state::txn_triggers` to `memtx_ddl_state::stmt_triggers` for consistency.

Reply:
Thanks, looks better! Done.